aspnetcore/signalr/messagepackhubprotocol.md

```diff
@@ -69,9 +72,10 @@ MessagePack support for the Javascript client is provided by the `@aspnet/signal
 npm install @aspnet/signalr-protocol-msgpack
 ```
 
-After installing the npm package, the module can be used directly via a JavaScript module loader or imported into the browser by referencing the *node_modules\\@aspnet\signalr-protocol-msgpack\dist\browser\signalr-protocol-msgpack.js* file. When using the MessagePack library via a browser `<script>` tag, the msgpack5 library must be referenced directly as well, it can be found in the *node_modules\msgpack5\dist\msgpack5.js* file.
+After installing the npm package, the module can be used directly via a JavaScript module loader or imported into the browser by referencing the *node_modules\\@aspnet\signalr-protocol-msgpack\dist\browser\signalr-protocol-msgpack.js* file. When using the MessagePack library via a browser `<script>` tag. The msgpack5 library must be referenced directly as well, it can be found in the *node_modules\msgpack5\dist\msgpack5.js* file.
```

Comment:
> When using the MessagePack library via a browser `<script>` tag. The msgpack5 library must be referenced directly as well, it can be found in the *node_modules\msgpack5\dist\msgpack5.js* file.

sentence fragment, change to something like

> Using MessagePack via a browser `<script>` tag requires a direct reference. The library can be found at *node_modules\msgpack5\dist\msgpack5.js*.

Reply:
Use a `<script>` tag to create a reference. The library can be found at *node_modules\msgpack5\dist\msgpack5.js.*

Via doesn't localize well, we don't use it according to the content guidelines.